Tests/SwiftParserTest/translated/SwitchTests.swift

```diff
@@ -950,7 +950,7 @@ final class SwitchTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected label in switch case")
+        DiagnosticSpec(message: "unexpected code '@moreGarbage' in switch case")
```

Comment:
Do we parse any single attribute rather than just `@unknown`?

Reply:
Yes, we do. That’s what the syntax tree currently allows.